item.rb

```diff
@@ -2,12 +2,13 @@
 
 class Item
   attr_reader :id, :archived
-  attr_accessor :publish_date
+  attr_accessor :publish_date, :genre
 
   def initialize(id, publish_date, archived: false)
     @id = id
     @publish_date = Date.parse(publish_date)
     @archived = archived
+    @genre = nil
```

Comment:
Kindly remove this instance variable, because according to the project requirements "properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created"

Reply:
Done! 
Thanks for your accuracy.